### PR TITLE
[Go]goev is a high-performance, lightweight, i/o event-driven network framework in Go.

### PR DESCRIPTION
**Pull request by shaovie**
This is my first attempt to submit a test, and I hope to pass it smoothly so that I can quickly see my framework in the first place on the leaderboard. ^_^

### CODE REVIEW

**Comment by shaovie** [comment]
This is my first attempt to submit a test, and I hope to pass it smoothly so that I can quickly see my framework in the first place on the leaderboard. ^_^

File: frameworks/Go/goev/src/go.mod [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module goev
+
+go 1.19
+
+require github.com/shaovie/goev latest
```
Comment (nbrady-techempower) [inline frameworks/Go/goev/src/go.mod:5]: Please point this to a specific commit, not latest

**Comment by shaovie** [comment]
I apologize for not submitting the go.sum file earlier, but I have now submitted it

**Comment by shaovie** [comment]
@nbrady-techempower Excuse me, can the code be merged?

**Comment by nbrady-techempower** [comment]
@shaovie Thank you. Still need to review. However, as it stands, this implementation should be marked as "Stripped" not "Realistic".

**Comment by shaovie** [comment]
Okay, thank you very much for your reply

**Comment by shaovie** [comment]
> @shaovie Thank you. Still need to review. However, as it stands, this implementation should be marked as "Stripped" not "Realistic".

@nbrady-techempower The framework has added support for parsing HTTP headers and parameter optimization.
thanks

**Comment by nbrady-techempower** [comment]
> > @shaovie Thank you. Still need to review. However, as it stands, this implementation should be marked as "Stripped" not "Realistic".
> 
> @nbrady-techempower The framework has added support for parsing HTTP headers and parameter optimization. thanks

I don't see any updates here that show that. Can you tell me where that was added?

File: frameworks/Go/goev/src/main.go
Comment (shaovie) [inline frameworks/Go/goev/src/main.go:1]: @nbrady-techempower Sorry, can you see me here? I just added splitting for HTTP headers and made a few parameter modifications. It seems like I can only be considered "stripped" now.

Comment (nbrady-techempower) [reply frameworks/Go/goev/src/main.go:1]: Yes, please update to "stripped"

Comment (alfiver) [reply frameworks/Go/goev/src/main.go:1]: alright

Comment (shaovie) [reply frameworks/Go/goev/src/main.go:1]: alright

**Pull request merged**